assets/queries/openAPI/invalid_content_type_for_multiple_files_upload/query.rego

```diff
@@ -0,0 +1,43 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+options := {"file", "filename"}
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	property := doc.paths[path][operation].requestBody.content[c].schema.properties[p]
+	p == options[x]
+
+	property.type == "array"
```

Comment:
Shouldn't we check array items' format? If it is not a binary, I think it can be a text, no?

Reply:
As I understand it, the 'type' and 'format' of 'items' define the type and format for each item. There is an example in https://swagger.io/docs/specification/describing-request-body/file-upload/.

Reply:
Yeap and what we want is binary items, check the colors example here https://swagger.io/docs/specification/describing-request-body/ on Form Data section, property type is array, but it is receiving an array of string colors, which will be encoded on the URL and not uploaded as a file...

Reply:
Thank you!! 😄  I will add this 'items' format check